csharp/Adapter/documentation/Mobius_API_Documentation.md

```diff
@@ -142,6 +142,13 @@
             Used for collect operation on RDD
```

Comment:
You could exclude changes in this file because you are not adding/removing any method from the public API